internal/hosts/hosts.go

```diff
@@ -141,20 +143,42 @@ func CreateFilterMetaQuery(query map[string][]string) string {
 	return strings.Join(filters, " and ")
 }
 
-func Load(client consul.Client, query_filter string, health_filter []string) (HostList, error) {
+func Load(client consul.Client, queryFilter string, healthFilter []string, tagsFilter []string) (HostList, error) {
 	var hosts = HostList{}
 
-	query := &consulApi.QueryOptions{Filter: query_filter}
-	consul_nodes, _, err := client.Catalog().Nodes(query)
+	query := &consulApi.QueryOptions{Filter: queryFilter}
```

Comment:
It's a pity that we cannot remove this filtering, but I guess that it is used to get the host list in the cluster and sap system pages